crates/routes/src/feeds.rs

```diff
@@ -213,7 +213,7 @@ fn get_feed_community(
   let community = Community::read_from_name(conn, community_name, false)?;
 
   let posts = PostQueryBuilder::create(conn)
-    .listing_type(ListingType::Community)
+    .listing_type(ListingType::All)
```

Comment:
I dont think you should have to specify ListingType at all in these cases. It should just default to `All` if nothing is specified. So `CommentQueryBuilder.listing_type` wouldnt be an option, but simply initialized with `ListingType::All`, and overwritten if this is called.

Reply:
Ah gotcha, ya I can do.